pyEpiabm/pyEpiabm/queue_sweep.py

```diff
@@ -0,0 +1,26 @@
+#
+# Sweeps for enqueued persons to update infection status
+#
+from .abstract_sweep import AbstractSweep
+from .infection_status import InfectionStatus
+
+
+class QueueSweep(AbstractSweep):
+    """Class to sweep through the enqueued persons
+    in each cell and update their infection status.
+
+    : param time: Simulation time, in days
```

Comment:
Sorry to be a pain (as I think the time parameter is labelled like this in a few places), but based on the new definitions added in the parameters file I don't think this is the case anymore! Time is indeed an integer, but you can now have multiple integer time steps per day so it is no longer measured in days. Sorry as I know this will need changing in a lot of places.

Reply:
Marked as unresolved as I don't think is resolved, whereas thought we said this should be changed?